root/etc/s6-overlay/s6-rc.d/init-dockwatch-config/run

```diff
@@ -6,4 +6,4 @@ s6-setuidgid abc php -r "require '/app/www/public/startup.php';"
 echo 's6-overlay init require(/app/www/public/startup.php) <-';
 
 # permissions
-lsiown -R abc /config
\ No newline at end of file
+lsiown -R abc:abc /config
```

Comment:
I think I'd rather see `chown -R abc /config` than force the group on these files. Whatcha think, nit?

Reply:
The original was yours, the change to just abc was your request as well. If you want it something else i am not going to pretend to know more about it than you nor say that i have taken the time to research it like i am sure you have at some point. If you agree with it, accept and merge it. If you want a change, request the change and have it tried how you are wanting it. You're the captain of that ship :)

Reply:
Couple questions to help me decide if `lsiown` is better for us than `chown`. 
- Does this run on every container start, or only once on container init? (I'm just not sure how the s6 stuff works)
- How many files are typically in `/config`? 
- What is the max file count someone may have in `/config`? 
  - If it's unlimited, we may have a performance problem to fix.

Reply:
I've decided this is probably the best approach:
```
find /config ! -user abc -exec echo chown abc {} + 
```

Reply:
That looks like it should achieve all the objectives (chown files and resolve error). 

To answer your first question, though, I believe it runs on every container start. Not sure on the file count average/max in ```/config```, however.

I can update my PR with the change, or you can modify it if you want. Up to you.

Reply:
Thank you for the info. I discussed this with an LSIO engineer and Austin, as well as researched the history of the `lsiown` script to come to this conclusion. 

Because: 1) We simply do not want to overwrite the group ownership, leaving control of that to the user. 2) `chown -R` can be slow if there's a lot of files, so we use `find` to speed it up a bit and avoid unnecessary disk-writes.

 Please update your contribution with the above command. Happy to accept it after that.

Reply:
@davidnewhall, I've updated the PR with the changes as requested.